app/package.json

```diff
@@ -5,13 +5,13 @@
   "dependencies": {
     "@aragon/api": "^2.0.0-beta.8",
     "@aragon/api-react": "^2.0.0-beta.8",
-    "@aragon/ui": "=1.0.0-alpha.22",
+    "@aragon/ui": "^1.3.0",
     "core-js": "^3.1.4",
     "react": "^16.8.6",
     "react-dom": "^16.8.6",
     "rxjs": "^6.5.2",
     "styled-components": "^4.3.2",
-    "date-fns": "2.0.0-alpha.22"
+    "date-fns": "^2.0.0-alpha.22"
```

Comment:
Could we fix this dependency version ? If i recall correctly a new version would give as issues